docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -9,35 +9,56 @@ Disease Description
 
 Lower respiratory infections(*LRI*), principally clinician-diagnosed pneumonia
 and bronchiolitis, is a major global killer of both children and adults. Symptoms
-include shortness of breath, weakness, fever, coughing and fatigue. It is mainly
+include shortness of breath, weakness, fever, coughing and fatigue. It is important to check for a fever. Symptoms can last about 7 days and the infection is contagious
+to others shortly before and while experiencing symptoms. It is mainly
 caused by four microorganisms - Streptococcus pneumoniae(*pneumococcal
 pneumonia*), Haemophilus influenzae type B(*Hib*),influenza, and respiratory
-syncytial virus(*RCV*). Pneumococcal pneumonia is the largest cause of LRI
-mortality.[Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
-
+syncytial virus(*RCV*). LRI can be caused by multiple pathogens and the pathogens may co-infect.
+Pneumococcal pneumonia is the largest cause of LRI
+mortality. [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
 Modeling LRI in GBD 2017
 ------------------------
 
+The GBD 2017 defined the time to recovery for LRI as an average of 10 days(*5-15 days*),
+which corresponds with a remission 36.5.
+
+
+GBD hierarchy
+-------------
+.. image:: lri_hierarchy.svg
```

Comment:
If you don't spell out what GBS stands for in the diagram itself, you should do so in the main text. It's probably also worth adding a line explaining what Guillan-Barre syndrome is, and in the limitations section, explain how ignoring it might affect our model.